play-scala-websocket-example/conf/application.conf

```diff
@@ -13,10 +13,14 @@ pekko {
   }
 }
 
-play.filters.enabled += play.filters.csp.CSPFilter
+# https://www.playframework.com/documentation/latest/SecurityHeaders
+# Allow URLs from the same origin to be loaded by frames and scripts
+play.filters.headers {
+  frameOptions = "SAMEORIGIN"
+}
```

Comment:
This is not needed. Or maybe I miss something?

Reply:
To be honest, I just copied configuration and another peace of code from Java example for two goal: fix this and sync with java example.
I did that during working on #666 without proper attention because the main target was add Gradle plugin. 
But as I see now, It was necessary to dive into more detail, sorry Matthias.

Reply:
I fully understand, no problem.